docs/external/forge/resources/managed-cache.md

```
@@ -0,0 +1,70 @@
+## [​](#settings) Settings
+The cluster’s “Settings” page allows you to manage the following:
+### [​](#configuration) Configuration
+You can update the compute size and high availability status of your cluster by clicking “Update configuration”. The cluster will enter a “Resizing” state during the update and will be temporarily unavailable.
+Downscaling compute is currently not supported. You may only increase the compute size of your cluster.
+### [​](#upgrade-window) Upgrade window
+Required upgrades, such as Valkey patches, are applied automatically during the configured maintenance window. You can set the preferred day of the week and hour (UTC) for these upgrades.
+### [​](#public-access) Public access
+By default, managed cache clusters only accept connections from resources within the same private network. Only [Laravel VPS](/docs/servers/laravel-vps) servers within the same Forge organization share this private network. Servers from other providers cannot connect privately, even if they are located in the same region. Enabling public access allows connections from any IP address. A confirmation prompt is displayed when enabling this setting.
+### [​](#deleting-a-cluster) Deleting a cluster
+You can permanently delete a cache cluster from the “Danger” section of the settings page. This action is irreversible. All data within the cluster will be permanently destroyed.
+## [​](#pricing) Pricing
```

markdownlint-cli2 (0.23.1)

[warning] Link fragments should be valid

(MD051, link-fragments)